sql/sql_select.cc

```diff
@@ -981,6 +1029,13 @@ JOIN::prepare(TABLE_LIST *tables_init,
   if (setup_fields(thd, ref_ptrs, fields_list, MARK_COLUMNS_READ,
                    &all_fields, 1))
     DBUG_RETURN(-1);
+  bool replaced_some_unversioned_field= false;
```

Comment:
Inconsistency: many terms for same meaning. Too long variable name. There is no sence in reading whole sentences just to read variable name.